src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$transitionSpeed: 300ms;
```

Comment:
transitionDuration sounds better for this property 😄

Reply:
you're right Nazar ;) noted!